cmd/appliance/stats.go

```diff
@@ -92,6 +96,17 @@ func statsNetworkPrettyPrint(n openapi.StatsAppliancesListAllOfNetwork) string {
 	return fmt.Sprintf("%s / %s", n.GetTxSpeed(), n.GetRxSpeed())
 }
 
+func statsDiskUsage(apiVersion int, stats openapi.StatsAppliancesListAllOfData) string {
+	if apiVersion < 18 {
+		return fmt.Sprintf("%g%%", stats.GetDisk())
+	}
+	diskInfo := stats.GetDiskInfo()
+	used, total := diskInfo.GetUsed(), diskInfo.GetTotal()
+	percentUsed := (used / total) * 100
+	r := fmt.Sprintf("%.2f%% (%s / %s)", percentUsed, appliancepkg.PrettyBytes(float64(used)), appliancepkg.PrettyBytes(float64(total)))
+	return r
+}
```

Comment:
:pick: You could remove apiVersion int here and reduce complexity by reversing it
```suggestion
func statsDiskUsage(stats openapi.StatsAppliancesListAllOfData) string {
	if v, ok := stats.GetDiskInfoOk(); ok {
		diskInfo := *v
		used, total := diskInfo.GetUsed(), diskInfo.GetTotal()
		percentUsed := (used / total) * 100
		return fmt.Sprintf("%.2f%% (%s / %s)", percentUsed, appliancepkg.PrettyBytes(float64(used)), appliancepkg.PrettyBytes(float64(total)))
	}
	return fmt.Sprintf("%g%%", stats.GetDisk())
}
```

Reply:
:+1: